torch/distributed/_state_dict_utils.py

```diff
@@ -19,6 +23,8 @@
     from torch.distributed.tensor._utils import compute_local_shape_and_global_offset
 
 
+logger = getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
We should remove this line. Users should be able to control the logging level, not the module.